examples/cluster/join_cluster_and_change_name.pp

```diff
@@ -0,0 +1,4 @@
+rabbitmq_cluster { 'test_cluster':
+  init_node => 'host1',
+}
+# This will join host2 to host1s cluster and set the cluster name to test_cluster
```

Comment:
nit, but would it make sense to put the comment above the example code vs. below?

Reply:
fixed + update REFERENCE.md

Reply:
Thanks! So, typically, we'd build reference.md using the rake task as part of the release process, but I can re-run it again (and deal with conflicts that come up) after merging this one.

I had been debating releasing 11.1 separately and having this ship in 11.2, but I'm Ok with putting this in 11.1 if you're confident about it.